cms/server/admin/handlers/dataset.py

```
@@ -856,3 +869,400 @@ def post(self, dataset_id):
+            sandbox.create_file_from_storage(exe_name,
+                                             generator.executable_digest,
+                                             executable=True)
+
+            cmd = ["./" + exe_name]
+            if language is not None:
+                try:
+                    cmds = language.get_evaluation_commands(exe_name)
+                    if cmds:
+                        cmd = cmds[0]
+                except Exception:
+                    pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)